src/cross_section.F90

```diff
@@ -563,6 +573,11 @@ subroutine calculate_urr_xs(i_nuclide, i_temp, E)
       micro_xs(i_nuclide) % fission = fission
       micro_xs(i_nuclide) % total = elastic + inelastic + capture + fission
 
+      ! Set the nu-photon production cross section
+      i_grid = int(log(E/energy_min_neutron)/log_spacing)
+      micro_xs(i_nuclide) % nu_photon_total = &
+           nuc % compute_nu_photon_total(E, i_temp, i_grid)
```

Comment:
Yeah, that's right it doesn't need to be recomputed again when computing urr xs with probability tables.